### ISSUE TITLE
Support for multiple Myos

### ISSUE THREAD

**Issue by McZonk** [McZonk/QCMyo#3]
I have only one Myo and the plug-in currently discards all further Myos.

What are the requirements for that?

My idea is using two Quartz Composer patches and have an arm input so you have a left arm and a right arm patch.

**Comment by FuzzBeast** [comment]
That is what I had in mind, but the patch needs some sort of (hopefully input controllable) way to tell them which myo is which. Oddly enough it seems to be defaulting (i.e. only communicating with) to the most recently connected Myo, (which means not the primary).

**Comment by FuzzBeast** [comment]
One of the returned commands is index, if this is controllable it should be possible to tell the patch which myo it should be communicating with, I'm looking through your xcode project, and while I don't know much to be of help, it seems that there should be a way to use the 'arm index' or 'arm name' fields to tell the patch which myo it needs to listen to.

**Comment by McZonk** [comment]
Right now the arm is only an output. But the idea would be to have an arm input and bind the accept Myo connection to a Myo that is used with this arm only.

**Comment by FuzzBeast** [comment]
Right, i guess that was what I was trying to say in a whole lot fewer words.

**Comment by FuzzBeast** [comment]
BTW If you need someone to test this out for you I would be willing.
